### PR TITLE
Feature/apikeeper 47

### PR DESCRIPTION
**Pull request by andrew--i**
apikeeper 47

### CODE REVIEW

File: apikeeper-provider/src/main/scala/com/apikeeper/oauth2/provider/config/Configuration.scala [hunk 1]
```diff
@@ -51,4 +54,11 @@ object Configuration {
       case NonFatal(_) => None
     }
   }
+
+  private def readValueWithDefault[T](v: => T, default: T) = {
```
Comment (igorkurilenko) [inline apikeeper-provider/src/main/scala/com/apikeeper/oauth2/provider/config/Configuration.scala]: setup default values in config

Comment (andrew--i) [inline apikeeper-provider/src/main/scala/com/apikeeper/oauth2/provider/config/Configuration.scala]: if there is no value in config?

Comment (igorkurilenko) [inline apikeeper-provider/src/main/scala/com/apikeeper/oauth2/provider/config/Configuration.scala]: Exception will be thrown and app won't start

Comment (andrew--i) [inline apikeeper-provider/src/main/scala/com/apikeeper/oauth2/provider/config/Configuration.scala]: ok

**Pull request merged**